components/__init__.py

```diff
@@ -9,6 +9,7 @@
     "DataField",
     "SpeedField",
     "CentrePower",
+    "Message",
```

Comment:
You're gonna have to import `Message` too. (`OverlayNew` won't even start right now lol)